src/core/cgroup.c

```diff
@@ -3095,7 +3102,7 @@ int unit_check_oom(Unit *u) {
                         LOG_UNIT_MESSAGE(u, "A process of this unit has been killed by the OOM killer."));
 
         if (UNIT_VTABLE(u)->notify_cgroup_oom)
-                UNIT_VTABLE(u)->notify_cgroup_oom(u);
+                UNIT_VTABLE(u)->notify_cgroup_oom(u, false);
```

Comment:
given that these two lines now appear at multiple places, maybe wrap them in a trivial helper. i.e. similar to unit_following() or unit_will_restart() or unit_kill(), which call the vtable method, but become NOPs if the method is NULL.

Reply:
Added `unit_notify_cgroup_oom()` wrapper